transformer_engine/pytorch/attention.py

```diff
@@ -5053,6 +5073,7 @@ def forward(
                         softmax_scale=self.softmax_scale,
                         causal="causal" in attn_mask_type,
                         deterministic=self.deterministic,
+                        **fa_optional_forward_kwargs_fp8,
```

Comment:
We should guard this behind a version check - is there already a published version of FA that contains your PR?

Reply:
There isn't. The latest release is 2.6.3 and my PR is just a commit after that on main. I can guardrail it with `>2.6.3`, but if people have used my method (`pip install "git+https://github.com/Dao-AILab/flash-attention.git#egg=flashattn-hopper&subdirectory=hopper"`) to install it, it doesn't matter what the base FA2 version is. That method just installs from the latest FA/hopper directory.

Reply:
Added try/except to deal with the default/custom descales in [0553a83](https://github.com/NVIDIA/TransformerEngine/pull/1174/commits/0553a8317f538129ee31086824336dd7f5924b16).